RFC-004-SWAP-Bitcoin.md

```diff
@@ -0,0 +1,125 @@
+# Bitcoin Ledger in SWAP Messages
+
+- RFC-Number: 004
+- Status: Draft
+- Discussion-issue: -
+- Created on: 1 Mar. 2019
+
+# Table of Contents
+<!-- markdown-toc start -->
+
+- [Description](#description)
+- [The Bitcoin Ledger](#the-bitcoin-ledger)
+    - [`network`](#network)
+- [The Bitcoin Asset](#the-bitcoin-asset)
+    - [`quantity`](#quantity)
+- [Registry extension](#registry-extension)
+    - [The Bitcoin Ledger](#the-bitcoin-ledger-1)
+        - [Bitcoin Networks](#bitcoin-networks)
+    - [The Bitcoin Asset](#the-bitcoin-asset-1)
+- [Examples](#examples)
+    - [SWAP Request](#swap-request)
+<!-- markdown-toc end -->
+
+## Description
+
+This RFC specifies how the Bitcoin blockchain and its native asset Bitcoin are described in [RFC002](./RFC-002-SWAP.md) SWAP messages.
+Bitcoin refers specifically to [Bitcoin Core](https://github.com/bitcoin/bitcoin/) and not any blockchains derived from it.
+As required by [RFC002](./RFC-002-SWAP.md), this RFC defines the *ledger* value `bitcoin` and its associated parameters as well as the *asset* `bitcoin`.
+This in combination with subsequent RFCs that define concrete execution steps for particular SWAP protocols enable users to execute and negotiate Bitcoin swaps.
```

Comment:
```suggestion
This in combination with subsequent RFCs that define concrete execution steps for particular swap protocols enable users to negotiate and execute Bitcoin swaps.
```

1. It is how it's referred in previous RFCs
2. to keep chronological order

Reply:
As far as I can tell we refer to `SWAP protocols` no `swap protocols` in previous PRs. Agreed on the other suggestions.

Reply:
Here we go: https://github.com/comit-network/RFCs/search?q=%22swap+protocol%22&unscoped_q=%22swap+protocol%22

Reply:
one of them uses SWAP protocol in the same sentence. The others are JSON keys. There is only one that uses "swap protocol" to mean "SWAP protocol". I'm going to say that it's the wrong one and this is right.